README.md

```diff
@@ -1,10 +1,11 @@
 # OSIsoft Cloud Services Python Library Sample
 
-**Version:** 0.5.3_preview
+**Version:** 0.6.0_preview
```

Comment:
thoughts on this being a 2nd number version change?  I feel like this doesn't break anything or add anything significant ... so I would just go to 5.4.

Reply:
I guess adding support for 3.7 is really not a 2nd number change, and everything else by definition is untouched. I'm fine switching it to a 3rd number change